src/app/services/xml-parsers/analogues-entries-parser.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { ParserRegister } from '.';
+import { Analogue, AnalogueClass, XMLElement } from '../../models/evt-models';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AnalogueEntriesParserService {
+
+  private tagName = `.${AnalogueClass}`;
```

Comment:
Tag names don't have the dot. The var name is incorrect. Change it or put the dot where needed.
Moreover those two props are private and used only once. You probably don't really need them, just use the value when needed.

Reply:
It's a css class because analogues and sources group up different tags